run/data/data.go

```diff
@@ -66,6 +67,7 @@ type Column struct {
 	UserDefined bool        // true if the type is user-defined
 	Nullable    bool        // true if the column is not NON NULL
 	HasDefault  bool        // true if the column has a default
+	PrimaryKey  bool        // true if the column is a primary key
```

Comment:
Can you make this HasPrimaryKey?  Otherwise it might be confusing if someone expects it to be the name of the primary key.

Reply:
I agree this should be changed. However, I think it should probably be `IsPrimaryKey`. Would that work for you?

Reply:
Yes, exactly right. It was late, and I was thinking this was on the table, not the column.   Speaking of which, can you put a method on Table called HasPrimaryKey that just does return len(t.PrimaryKeys) > 0 . ? I think that'll be something people will want to use fairly often.